book/06-github/sections/2-contributing.asc

```diff
@@ -1,78 +1,78 @@
-=== Contributing to a Project
+=== Pag-aambag sa isang Proyekto
 
-Now that our account is set up, let's walk through some details that could be useful in helping you contribute to an existing project.
```

Comment:
kindly change na-setup to na-set up ? 

As setup is a noun and set up is a verb :)